### PR TITLE
fix(module:image): image fit content when drag released

### PR DESCRIPTION
**Pull request by stygian-desolator**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

image will fit content when drag released

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by stygian-desolator** [comment]
@hsuanxyz  cc, thanks~

**Comment by stygian-desolator** [comment]
wait

**Comment by stygian-desolator** [comment]
@hsuanxyz cc, thanks~

File: components/image/utils.ts [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+ * Height   |                                      |
+ * |        |                                      |
+ * |        |                                      |
+ * |        height                                 |
+ * |        |                                      |
+ * ---------|                                      |
+ *          |                                      |
+ *          |                                      |
+ *          |                                      |
+ *          ----------------------------------------
```
Comment (hsuanxyz) [inline components/image/utils.ts:46]: Nice comments. BTW, you can use http://asciiflow.com/.

File: components/image/utils.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  }
+
+  return fixPos;
+}
+
+export function getOffset(node: HTMLElement): { left: number; top: number } {
+  const box = node.getBoundingClientRect();
+  const docElem = document.documentElement;
+
+  // < ie8 不支持 win.pageXOffset, 则使用 docElem.scrollLeft
```
Comment (hsuanxyz) [inline components/image/utils.ts:91]: We only support to 11 for IE.

File: components/image/utils.ts (hunk 1 above)
Comment (stygian-desolator) [reply components/image/utils.ts:47]: Thanks for recommending this tool !
I added a link about our discussion.
https://github.com/NG-ZORRO/ng-zorro-antd/pull/6262/files#diff-b07457ff5af807a99da47941e3445a149e907419c383c67c865a7200cc063ec8R7

File: components/image/utils.ts (hunk 2 above)
Comment (stygian-desolator) [reply components/image/utils.ts]: Got, I updated this comment

**Comment by stygian-desolator** [comment]
Please review again, thanks~
But I'm a little confused, why corecov/patch failed 🤔

**Review by hsuanxyz — APPROVED** [review]
LGTM, The codecov's baseline may be broken, you don't have to worry, it doesn't stop us from merge it .

**Pull request merged**